src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -126,6 +126,15 @@
   <data name="ApprovedCipherModeMessage" xml:space="preserve">
     <value>It uses an unsafe Cipher Mode {0}</value>
   </data>
+  <data name="DoNotAddSchemaByURL" xml:space="preserve">
+    <value>Do Not Add Schema By URL</value>
+  </data>
+  <data name="DoNotAddSchemaByURLDescription" xml:space="preserve">
+    <value>This overload of XmlSchemaCollection.Add method is potentially unsafe because it may load untrusted XML which includes dangerous external references. Either use an XmlSecureResolver or disable DTD processing.</value>
```

Comment:
Is there a way for people to disable DTD processing or specify an XML resolver with XmlSchemaCollection.Add(string, string)?  I vote we just get rid of the description entirely.